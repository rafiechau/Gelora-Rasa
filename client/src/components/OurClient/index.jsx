import { FormattedMessage } from 'react-intl';
import classes from './style.module.scss';

export const OurClient = () => (
  <div className={classes.containerLogo} data-testid="our-client">
    <p className={classes.titleContainerLogo}>
      <FormattedMessage id="app_events_description" />
    </p>
    <div className={classes.containerBigCompany}>
      <img src="assets/images/apple-111.svg" alt="semina" />
      <img src="assets/images/Adobe.svg" alt="semina" />
      <img src="assets/images/slack-21.svg" alt="semina" />
      <img src="assets/images/spotify-11.svg" alt="semina" />
      <img src="assets/images/google-2015.svg" alt="semina" />
    </div>
  </div>
);
