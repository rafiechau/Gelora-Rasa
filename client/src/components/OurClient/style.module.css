.containerLogo {
  margin-top: 2rem;
}
.containerLogo p {
  text-align: center;
  font-weight: 500;
  font-size: 16px;
  color: #A3A5AA;
}
.containerLogo .containerBigCompany {
  display: flex;
  flex-wrap: wrap;
  justify-content: center;
  align-items: center;
  gap: 1.5rem;
  padding-top: 1rem;
}

@media (max-width: 768px) {
  .containerLogo .titleContainerLogo, .containerLogo .containerBigCompany {
    font-size: 14px;
    gap: 1rem;
  }
}
@media (max-width: 480px) {
  .containerLogo .titleContainerLogo {
    font-size: 12px;
  }
  .containerLogo .containerBigCompany {
    gap: 3rem;
  }
}/*# sourceMappingURL=style.module.css.map */