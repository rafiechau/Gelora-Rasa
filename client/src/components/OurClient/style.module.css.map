{"version":3,"sources":["style.module.scss","style.module.css"],"names":[],"mappings":"AAAA;EACI,gBAAA;ACCJ;ADAI;EACI,kBAAA;EACA,gBAAA;EACA,eAAA;EACA,cAAA;ACER;ADAI;EACI,aAAA;EACA,eAAA;EACA,uBAAA;EACA,mBAAA;EACA,WAAA;EACA,iBAAA;ACER;;ADEA;EAEQ;IACI,eAAA;IACA,SAAA;ECAV;AACF;ADKA;EAEQ;IACI,eAAA;ECJV;EDOM;IACI,SAAA;ECLV;AACF","file":"style.module.css"}