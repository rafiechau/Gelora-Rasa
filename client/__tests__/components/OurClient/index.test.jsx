// OurClient.test.jsx
import React from 'react';
import { render, screen } from '@testing-library/react';
import { IntlProvider } from 'react-intl';
import { OurClient } from '@components/OurClient';

// Lokalisasi dan pesan contoh (sesuaikan dengan kebutuhan Anda)
const messages = {
  app_events_description: 'Deskripsi Acara',
};

describe('OurClient Component', () => {
  beforeEach(() => {
    render(
      <IntlProvider locale="id-ID" messages={messages}>
        <OurClient data-testid="our-client" />
      </IntlProvider>
    );
  });

  it('should render correctly', () => {
    expect(screen.getByTestId('our-client')).toBeInTheDocument();
  });

  it('should match snapshot', () => {
    const ourClientComponent = screen.getByTestId('our-client');
    expect(ourClientComponent).toMatchSnapshot();
  });
});
