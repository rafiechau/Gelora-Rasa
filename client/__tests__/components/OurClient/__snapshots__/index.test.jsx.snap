// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OurClient Component should match snapshot 1`] = `
<div
  class="containerLogo"
  data-testid="our-client"
>
  <p
    class="titleContainerLogo"
  >
    Deskripsi Acara
  </p>
  <div
    class="containerBigCompany"
  >
    <img
      alt="semina"
      src="assets/images/apple-111.svg"
    />
    <img
      alt="semina"
      src="assets/images/Adobe.svg"
    />
    <img
      alt="semina"
      src="assets/images/slack-21.svg"
    />
    <img
      alt="semina"
      src="assets/images/spotify-11.svg"
    />
    <img
      alt="semina"
      src="assets/images/google-2015.svg"
    />
  </div>
</div>
`;
